pkg/dataflatten/json.go

```diff
@@ -4,13 +4,26 @@ import (
 	"encoding/json"
 	"fmt"
 	"os"
+
+	"golang.org/x/text/encoding/unicode"
+	"golang.org/x/text/transform"
 )
 
 func JsonFile(file string, opts ...FlattenOpts) ([]Row, error) {
 	rawdata, err := os.ReadFile(file)
 	if err != nil {
 		return nil, err
 	}
+
+	// Attempt to decode utf16 data.
+	valid_json := json.Valid(rawdata)
+	if !valid_json {
+		rawdata, _, err = transform.Bytes(unicode.UTF16(unicode.LittleEndian, unicode.UseBOM).NewDecoder(), rawdata)
+		if err != nil {
+			return nil, err
```

Comment:
I think this may end up as the wrong error. We probably want to return something like "not json" and not whatever the transform error is

Reply:
I tweaked your errors. Lets see if I got the go code right in a PR suggest window 😆